packages/bdd-tests/src/configs/api.ts

```diff
@@ -0,0 +1,54 @@
+import axios from "axios";
+import fs from "fs";
+const genid = require("../../util/genId");
+
+let url = process.env.base_url;
+let dir = './resources';
+
+let instance = axios.create({
+  baseURL: url,
+  headers: {
+    "NHSD-Session-URID": "555254242106",
+    "X-Request-ID": genid.generateRandomUUID(),
+    "X-Correlation-ID": "11C46F5F-CDEF-4865-94B2-0EE0EDCC26DA",
```

Comment:
This shouldn't be hard coded, it should either be generated as above or not provided

Reply:
Agreed